serverless/index.asciidoc

```diff
@@ -11,7 +11,6 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 :sec-badge: <<what-is-security-serverless,image:images/sec-badge.svg[Security]>>
 
 // The values of these attributes are different in stateful vs serverless
-:es: Elasticsearch Serverless
```

Comment:
This is the important change for Elasticsearch writers, to continue to be able to use ye olde `{es}` for plain olde Elasticsearch